src/vsphere_cpi/spec/unit/cloud/vsphere/selection_pipeline_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'spec_helper'
+require 'cloud/vsphere/selection_pipeline'
+
+describe VSphereCloud::SelectionPipeline do
+  subject { described_class.new(criteria) { placement } }
+  let(:criteria) { :fake_criteria }
+  let(:placement) { [:fake_placement] }
+
+  class Integer
```

Comment:
???

Reply:
Logging is using inspect method to print out certain messages. Inspect is not implemented for the for the fake integer placements we have. That's why I just introduced a dummy one here.